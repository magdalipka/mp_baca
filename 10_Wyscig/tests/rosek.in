2
3 4
Batman 10 1 3 4
Listonosz 0 1 2 3
Klocuch 2 3 4 5
4 3
a1 1 0 1
a2 1 0 2
a3 1 1 -1
a4 1 0 3
