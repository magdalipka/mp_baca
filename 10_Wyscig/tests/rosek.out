Batman - 1 pkt.
Listonosz - 0 pkt.
Klocuch - 3 pkt.
a1 - 0 pkt.
a2 - 0 pkt.
a3 - 1 pkt.
a4 - 0 pkt.
